x-pack/plugins/security_solution/public/cases/components/tag_list/index.tsx

```diff
@@ -103,7 +103,7 @@ export const TagList = React.memo(
             {tags.length > 0 &&
               !isEditTags &&
               tags.map((tag, key) => (
```

Comment:
is `tags` an array? if so please don't use `index`(key) for for the `key` value
https://reactjs.org/docs/lists-and-keys.html#keys